subprojects/code-quality/src/main/groovy/org/gradle/api/plugins/quality/CodeNarcPlugin.java

```diff
@@ -37,7 +37,7 @@
  */
 public class CodeNarcPlugin extends AbstractCodeQualityPlugin<CodeNarc> {
 
-    public static final String DEFAULT_CODENARC_VERSION = "1.6.1";
+    public static final String DEFAULT_CODENARC_VERSION = "2.0.0";
```

Comment:
When we make big jumps like this, we should also revisit CodenarcCoverage to see if we need to keep around the old version.  I would probably drop 1.2.1 and replace it with 1.6.1.

Reply:
Thanks for catching